andela/heimdal-ah-backend
# #162171032 implement article readtime

#### What does this PR do?
- This PR adds functionality to calculate the average length of time required to read an article.

#### Description of Task to be completed?
- When an article is created, the reading time of the article should be estimated from the content of the article.
- When users fetch an article, the reading time should also be displayed.

#### How should this be manually tested?
- Clone this Repo
- ```cd``` in the repo
- Run ```npm install``` to install all the dependencies
- Run ```npm run start:dev``` to kick-start the local server in development mode.
- Signup a new user through the signup endpoint with postman.
- Copy out the token returned.
- With the token returned, visit the article creation endpoint,and add the token to the headers on postman (access-token).
- Create the new article supplying the body, title, image, description as well as all required parameters as stated above.
- Watch and confirm that the response from the request contains the average reading time of the article.

#### What are the relevant pivotal tracker stories?
[#162171032](https://www.pivotaltracker.com/story/show/162171032)

#### Any background context you want to add?
- None

#### Screenshots
